src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLambdaExpressions.java

```diff
@@ -14,16 +14,17 @@
         String.valueOf("Hello world one!");
     };
 
-    static Runnable r2 = () ->String.valueOf("Hello world two!"); // violation
+    static Runnable r2 = () ->String.valueOf(""); // violation ''->' is not followed by whitespace'
 
-    static Runnable r3 = () ->{String.valueOf("Hello world two!");}; // violation
+    static Runnable r3 = () ->{String.valueOf("");}; // violation ''->' is not .* whitespace'
```

Comment:
Remove static keyword and use full message.

Reply:
Done

Reply:
Resolved.